modules/virt-deploying-operator-cli.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-virt-cli.adoc
+
+[id="virt-deploying-operator-cli_{context}"]
+= Deploying the {VirtProductName} Operator by using the CLI
+
+You can deploy the {VirtProductName} Operator by using the `oc` CLI.
+
+.Prerequisites
+
+* An active subscription to the {VirtProductName} catalog in the `openshift-cnv` namespace.
+
+.Procedure
+
+. Create a file named `hco_cr.yaml` by running the following command:
++
+----
+$ touch hco_cr.yaml
+----
+
+. Open `hco_cr.yaml` in a text editor and paste the following manifest into the file:
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1alpha1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: openshift-cnv
+spec:
+  BareMetalPlatform: true
+----
+
+. Save `hco_cr.yaml` and exit the text editor.
+
+. Deploy the {VirtProductName} Operator by running the following command:
++
+----
+$ oc apply -f hco_cr.yaml
+----
+
+.Verification steps
+
+* Ensure that {VirtProductName} deployed successfully by checking for running {VirtProductName} Pods. Run the following command to view all Pods in the `openshift-cnv` namespace:
```

Comment:
It's not enough that all the pods are just running in order to ensure successful installation.
The verification steps are:
1. All pods are running *and* ready (i.e. `1/1` in `READY` column)
2. HCO Custom Resource condition "Available" is "True", can be verified by:
```
$ oc wait hyperconverged kubevirt-hyperconverged --for condition=Available -n openshift-cnv
hyperconverged.hco.kubevirt.io/kubevirt-hyperconverged condition met
```

Reply:
Another option is to verify the phase of the CSV is "Succeeded":
```
$ oc get csv -n openshift-cnv
NAME                                      DISPLAY                    VERSION   REPLACES   PHASE
kubevirt-hyperconverged-operator.v2.4.0   OpenShift virtualization   2.4.0                Succeeded
```

Reply:
Thanks, @orenc1. I opted to include the CSV step for simplicity, and I used document attributes for the version and display name (I assume that the display name should be OpenShift Virtualization; if it isn't, that seems like a bug).

Preview build: http://file.bos.redhat.com/pousley/070920/cnv-cli-deploy/virt/install/installing-virt-cli.html

I'm open to more SME feedback if you have it, but I'll move the Jira to QE.

Reply:
> I assume that the display name should be OpenShift Virtualization
Correct, that was fixed downstream recently.